src/main/java/io/gatling/mojo/VerifyMojo.java

```diff
@@ -53,12 +54,19 @@ private void verifyLastRun() throws IOException, MojoFailureException, MojoExecu
 
         if (results.toFile().exists()) {
             for (String line : Files.readAllLines(results)) {
+                checkError(line);
                 File directory = new File(resultsFolder, line);
                 searchForAssertionFailures(directory);
             }
         }
     }
 
+    private void checkError(String line) throws MojoFailureException {
+        if (StringUtils.contains(line, LAST_RUN_FILE_ERROR_LINE)) {
+            throwFailureException("Gatling had errors during execution!");
```

Comment:
You're losing the valuable information message that was logged in this line. You should be chopping the line marker and propagate the actual error message.